### DISCUSSION TITLE
Getting error in installation

### DISCUSSION THREAD

**Discussion by ritamgit-alt** [fury-gl/fury#746] [General]
## Description
Basically when I'm trying to install fury from the source or trying to test the installation, I'm getting a file error.

## Way to reproduce

```
pip install -r requirements/default.txt
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements/default.txt'
```

`{'fury_version': '0.8.0', 'pkg_path': '/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/fury', 'commit_hash': 'b937ff3418e7c64b0fa89921a54c083ff78dc055', 'sys_version': '3.11.1 (v3.11.1:a7a450f84a, Dec  6 2022, 15:24:06) [Clang 13.0.0 (clang-1300.0.29.30)]', 'sys_executable': '/Library/Frameworks/Python.framework/Versions/3.11/bin/python3', 'sys_platform': 'darwin', 'numpy_version': '1.24.1', 'scipy_version': '1.10.1', 'vtk_version': '9.2.6', 'matplotlib_version': '3.6.3', 'dipy_version': '1.6.0'}
`

**Comment by skoudoro** [comment]
it just means that you are in the wrong folder for the installation

**Comment by ritamgit-alt** [comment]
thanks for hitting me up, I was just so fool.
